books/free-programming-books-langs.md

```diff
@@ -1130,6 +1137,9 @@ That section got so big, we decided to split it into its own file, the [BY SUBJE
 * [JavaScript Spessore](https://web.archive.org/web/20160325064800/https://leanpub.com/javascript-spessore/read) - Reginald Braithwaite (HTML)
 * [JavaScript Succinctly](https://www.syncfusion.com/resources/techportal/ebooks/javascript) - Cody Lindley (PDF, Kindle; email address *requested*, not required)
 * [JavaScript the Right Way](https://github.com/braziljs/js-the-right-way) - William Oliveira, Allan Esquina (HTML)
+* [Javascript Tutorial](https://www.w3schools.com/js)
```

Comment:
We can move this to interactive tutorials as the user can interact with the code and run it

Reply:
```suggestion
```
already there